introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I just want to get going with the tutorial, but it's complaining the introduction is too short. I hope this will do.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.